### ISSUE TITLE
Latest setup.sh checkin breaks gcc-arm installation

### ISSUE THREAD

**Issue by verderog** [TexasInstruments/edgeai-tidl-tools#72]
@vtrip97 Lines 428/429 of your checkin of setup.sh do not reset the path and cause gcc-arm and cl7x which follow it to get installed to an unexpected location.

**Comment by verderog** [comment]
Quick fix, add `cd -` as shown below:

```
# graph optimizer tool setup
if [[ $arch == x86_64 && $skip_model_optimizer -eq 0 ]]; then
    cd $SCRIPTDIR/scripts/osrt_model_tools/onnx_tools/tidl-onnx-model-optimizer
    source ./setup.sh
    cd -
fi

```

**Comment by verderog** [comment]
Fixed in a8b62829572fe24af1d68e7c940474488aedfce1
